drivers/mpu6050/MPU6050.hpp

```diff
@@ -56,7 +56,8 @@ class MPU6050: public ImuSensor
 	MPU6050(const char *device_path) :
 		ImuSensor(device_path, MPU6050_MEASURE_INTERVAL_US, false), // false = sensor has no mag
 		_last_temp_c(0.0f),
-		_temp_initialized(false)
+		_temp_initialized(false),
+		_packets_per_cycle_filtered(1.0)
```

Comment:
Help me out here, what's the sampling frequency again? Are you using the FIFO buffer at all actually?